config/redis.yml

```diff
@@ -190,6 +190,9 @@ development: &defaults
   vanotify_confirmation_email_store:
     namespace: confirmation_email
     each_ttl: 600 # 10 minutes
+  va_profile_veteran_status:
+    namespace: va_profile_veteran_status
+    each_ttl: 86400
```

Comment:
How did you decide on this value?

Reply:
86400 is the equivalent of 24 hours. Figured veteran status is unlikely to change more than once every day and noticed the 86400 convention set for prior caches like emis_response.

Reply:
Nice. Sounds good to me.